src/log.cr

```diff
@@ -66,9 +66,23 @@
 # end
 # ```
 #
+# ### Default logging configuration
+#
+# By default entries from all sources with `Info` and above severity will
+# be logged to `STDOUT` using the `Log::IOBackend`.
+#
+# If you need to change the default level, backend or sources call `Log.setup` upon startup.
+#
+# ```
+# Log.setup(:debug)                     # Log debug and above for all sources to STDOUT
+# Log.setup("myapp.*, http.*", :notice) # Log notice and above for myapp.* and http.* sources only
```

Comment:
Perhaps "Log notice and above for myapp.* and http.* sources, and log nothing for any other source" to be extra clear.